src/utils/navigation.ts

```diff
@@ -52,6 +52,11 @@ export const NAVIGATION: Navigation = {
   [config.WHO_IS_MAKING_UPDATE_URL]: {
     currentPage: config.WHO_IS_MAKING_UPDATE_PAGE,
     previousPage: () => config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
+    nextPage: [config.OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE] // TO DO: UAR-102 needs added
```

Comment:
Need to make this

nextPage: [config. UPDATE_DUE_DILIGENCE_PAGE, config.OVERSEAS_ENITY_DUE_DILIGENCE_PAGE]

whenever UAR-102 is merged